source/grammar/qasm3.g4

```diff
@@ -340,6 +360,7 @@ expressionTerminator
     | Integer
     | RealNumber
     | Identifier
+    | setDeclaration
```

Comment:
I think this needs to be `membershipTest`. I don't think `{1,2,3}` is an expression on its own. Should be the boolean result of a measurement test

Reply:
`{1,2,3}` looks an array. Though current spec does not say any about array-type, initialization of its variables will be this form.

Reply:
This has been fixed.